tasks/README.md

```diff
@@ -173,13 +173,19 @@ by the webhook which automatically starts a release runner.
 
 ### Automated testing
 
-When a pull request event is received, the webhook will scan the pull request
-for tests that need to be run. These will be put in a distributed queue to later
-be consumed by other task bots. If "[no-test]" is in the title of the pull
+When a pull request event is received, the webhook will publish the event to a
+distributed queue. These events are then consumed and interpreted by task bots
```

Comment:
As that part will be the same for any webhook event, I think these two sentences should be factored out into a first paragraph by themselves. "a distributed queue" should be more specific, like "to the webhook AMQP queue".

Reply:
ack!